--- /home/gh/python/lora/trackerd/repo/Example/LoRaWAN/examples/TrackerD/common.h	2026-08-15 07:17:35.470523090 +0200
+++ src/common.h	2026-08-23 20:53:14.946715414 +0200
@@ -51,7 +51,7 @@
     void GPSDATA_CLEAR(void);
     void customize_freq_set(void);
     void Band_information(void);
-    uint8_t     gps_data_buff[15];
+    uint8_t     gps_data_buff[17];   /* 15 (sensor_type 13) bzw. 17 (22) */
     char        blemask_data[12]={'0','0','0','0','0','0'};
     char        wifimask_data[12]={'0','0','0','0','0','0'};
     char        buff1[100];
